scripts/dev-e2e.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+set -euo pipefail
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+STATUSFILE='armadactl-query.txt'
+AOREPO='https://github.com/armadaproject/armada-operator.git'
+ARMADA_CTL_VERSION='0.18.3'
+JOB_DETAILS=1
+
+usage() {
+  cat > /dev/stderr  <<USAGE
+Usage: ${0} -s
+  -s => get job status details from Armada
+
+USAGE
+  exit 1
+}
+
+while getopts "s" opt; do
+    case "${opt}" in
+        s )
+          JOB_DETAILS=1 ;;
+        * )
+          echo "opt is $opt"
+          usage ;;
+    esac
+done
+
+scripts="$(cd "$(dirname "$0")"; pwd)"
+source "$scripts"/init.sh
+
+AOHOME="$scripts/../../armada-operator"
```

Comment:
I'd prefer the definition of such variables in one place, first thing in the scrip. So rather move definition of `scripts` up than moving `AOHOME` down here.

Reply:
Fixed in commit 704accd - I moved this up, and also tweaked the ordering (the color definitions are now below the vars that a user might want to tweak), renamed `ARMADA_CTL_VERSION` to `ARMADACTL_VERSION` to be consistent with the actual command.